### ISSUE TITLE
packaged-as-an-iexpress-installer

### ISSUE THREAD

**Issue by recvfrom** [fireeye/capa-rules#185]
## Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked that your rule idea isn't already filed: [search](https://github.com/fireeye/capa-rules/issues?q=is%3Aissue+is%3Aopen+)

## Summary

It'd be cool to have a rule that detects EXEs created with `iexpress.exe`

## Examples

52e25bdd600695cfed0d4ee3aca4f121bfebf0de889593e6ba06282845cf39ea

## Features

Is it possible to match on the EXE manifest with capa?
```
<!-- Copyright (c) Microsoft Corporation -->
<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">

  <assemblyIdentity version="5.1.0.0"
     processorArchitecture="amd64"
     name="wextract"
     type="win32"/>
  
  <description>IExpress extraction tool</description>
...
```

Otherwise, matching on the two strings shown below in the same function scope should be a good indicator:
![image](https://user-images.githubusercontent.com/36489577/99586273-7ee9c100-29b5-11eb-841f-fb3062c4f3a2.png)

From: https://analyze.intezer.com/analyses/6e1d2029-a4ac-477a-8187-03670732b745/families/0a29c6e3-2b86-4e82-b034-57a08b011ad6

## Additional context

N/A

### Namespace
executable/installer/iexpress

### References
https://en.wikipedia.org/wiki/IExpress

### Other rule meta information
T1207 - Obfuscated Files or Information

**Comment by mr-tz** [comment]
Nice, we can match on the manifest strings and the other `(w)extract` related strings.

**Comment by williballenthin** [comment]
@recvfrom  you've had some great rules in the past, and we really appreciate the documentation you put together in these issues! please also feel free to directly open PRs (if you want to draft the rules, too). minor benefit is that there'd be less latency in you waiting for us to say "good idea go ahead!"

**Comment by recvfrom** [comment]
@williballenthin thank you! I recently discovered that my work wants us to track work-related contributions to open source projects through a new system they created, and I haven't had a chance to figure out what the full process for that is.  I figured I'd open tickets in the meantime, though, so I don't move on to other things and forget about it. I do plan to circle back to these at some point, though, unless someone else gets to it first.
